### PR TITLE
docs: Translate to Japanese

### PR DESCRIPTION
**Pull request by RyoTagami**
# Summary

Document translated into Japanese.

## Other Information

Translated from the English document as of fce6c534daf6c0b2eed7dc633b18f7b87f665d2f .

### CODE REVIEW

**Comment by hyperupcall** [comment]
thanks! someone with permissions will need to run the workflows

File: docs/ja-jp/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+features:
+  - title: 単一ツール
+    details: "単体のCLIツールとコマンドインターフェースで、各プロジェクトのランタイムを管理できます。"
+    icon: 🎉
+  - title: プラグイン
+    details: "既存ランタイム・ツールを使用した大規模なエコシステムです。必要に応じて新しいツールをサポートできるシンプルなAPIを用意しています!"
+    icon: 🔌
+  - title: 後方互換性
+    details: ".nvmrc、.node-versions、.ruby-versionといった既存構成ファイルから、スムーズに移行できます!"
```
Comment (kodaka) [inline docs/ja-jp/index.md:28]:
```suggestion
    details: ".nvmrc、.node-version、.ruby-versionといった既存構成ファイルから、スムーズに移行できます!"
```

Please check https://github.com/asdf-vm/asdf/pull/1679

**Review by kodaka — COMMENTED** [review]
I did not check Japanese, but my suggestion was included as I expected (https://github.com/asdf-vm/asdf/pull/1667/commits/5a3e577202f0bf968a24237c4f50fb1faf0b3707)
Thank you!

**Review by jthegedus — APPROVED** [review]
Looks great, thanks!

**Pull request merged**